<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendsDialog</class>
 <widget class="QWidget" name="FriendsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>491</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="titleBarFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="titleBarPixmap">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/png/network2.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="StyledLabel" name="titleBarLabel">
        <property name="text">
         <string>Network</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>123</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="helpButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/help_64.png</normaloff>:/icons/help_64.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QFrame" name="headFrame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="0" column="0" rowspan="2">
           <widget class="AvatarWidget" name="avatar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QToolButton" name="mypersonalstatusLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Edit status message</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/images/edit_16.png</normaloff>:/images/edit_16.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="StyledElidedLabel" name="nicknameLabel">
            <property name="text">
             <string notr="true">Nickname (Location)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="NewFriendList" name="friendList" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="ChatTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="groupChatTab">
       <attribute name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/png/chats.png</normaloff>:/icons/png/chats.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Broadcast</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="ChatWidget" name="chatWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionClear_Chat_History">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-clear-history.png</normaloff>:/images/edit-clear-history.png</iconset>
   </property>
   <property name="text">
    <string>Clear Chat History</string>
   </property>
  </action>
  <action name="actionAdd_Friend">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/user/add_user16.png</normaloff>:/images/user/add_user16.png</iconset>
   </property>
   <property name="text">
    <string>Add Friend</string>
   </property>
  </action>
  <action name="actionSet_your_Avatar">
   <property name="icon">
    <iconset>
     <normaloff>:/images/add_image24.png</normaloff>:/images/add_image24.png</iconset>
   </property>
   <property name="text">
    <string>Add your Avatar Picture</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionSet_your_Personal_Message">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/message-news.png</normaloff>:/images/message-news.png</iconset>
   </property>
   <property name="text">
    <string>Set your status message</string>
   </property>
   <property name="toolTip">
    <string>Edit your status message</string>
   </property>
  </action>
  <action name="actionMessageHistory">
   <property name="text">
    <string>Browse Message History</string>
   </property>
   <property name="toolTip">
    <string>Browse History</string>
   </property>
  </action>
  <action name="actionSave_History">
   <property name="text">
    <string>Save Chat History</string>
   </property>
   <property name="toolTip">
    <string>Save Chat History</string>
   </property>
  </action>
  <action name="actionAdd_Group">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/user/add_group22.png</normaloff>:/images/user/add_group22.png</iconset>
   </property>
   <property name="text">
    <string>Add a new Group</string>
   </property>
   <property name="toolTip">
    <string>Add a new Group</string>
   </property>
  </action>
  <action name="actionDelete_Chat_History">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit-clear-history.png</normaloff>:/images/edit-clear-history.png</iconset>
   </property>
   <property name="text">
    <string>Delete Chat History</string>
   </property>
   <property name="toolTip">
    <string>Deletes all stored and displayed chat history</string>
   </property>
  </action>
  <action name="actionCreate_new_Chat_lobby">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/user/agt_forum24.png</normaloff>:/images/user/agt_forum24.png</iconset>
   </property>
   <property name="text">
    <string>Create new Chat lobby</string>
   </property>
   <property name="toolTip">
    <string>Create new Chat lobby</string>
   </property>
  </action>
  <action name="actionChooseFont">
   <property name="text">
    <string>Choose Font</string>
   </property>
  </action>
  <action name="actionResetFont">
   <property name="text">
    <string>Reset font to default</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AvatarWidget</class>
   <extends>QLabel</extends>
   <header>gui/common/AvatarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StyledLabel</class>
   <extends>QLabel</extends>
   <header>gui/common/StyledLabel.h</header>
  </customwidget>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/chat/ChatWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">gui/chat/ChatTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StyledElidedLabel</class>
   <extends>QLabel</extends>
   <header>gui/common/StyledElidedLabel.h</header>
  </customwidget>
  <customwidget>
   <class>NewFriendList</class>
   <extends>QWidget</extends>
   <header>gui/common/NewFriendList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
